enterprise/app/workflows/workflows.css

```diff
@@ -250,7 +250,7 @@
 .workflows-github-import .install-app-banner .banner-content {
   display: flex;
   flex-direction: column;
-  gap: 8px;
+  gap: 14px;
```

Comment:
mind posting a screenshot of the new banner? also could we use 16px here? Generally our spacing values are multiples of 8px